tensorflow/python/ops/nn_grad.py

```diff
@@ -364,7 +364,7 @@ def _ReluGrad(op, grad):
 @ops.RegisterGradient("EluGrad")
 def _EluGradGrad(op, grad):
   elu_x = op.inputs[1]
-  return (gen_nn_ops.elu_grad(grad, op.outputs[0]),
+  return (gen_nn_ops.elu_grad(grad, elu_x),
```

Comment:
Good catch! 

Do the test here pass both before and after this change, even with gradient_checker? 

https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/kernel_tests/relu_op_test.py#L255

If so, we need to patch up the test coverage hole. Thanks!

Reply:
Thanks for following up. I don't have the time right now, but can make the changes in a week or two.